.github/workflows/buildTestBinaries.yml

```diff
@@ -0,0 +1,40 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Build test binaries
+
+# Controls when the action will run. Triggers the workflow push is only on main branch and PR on any branch.
+on:
+  pull_request:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build-binaries:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+      
+      - uses: actions/setup-go@v3
+        with:
+          go-version: "^1.18" # The Go version to download (if necessary) and use.
+
+      
+      - name: Build client
+        uses: goreleaser/goreleaser-action@v3
+        with:
+          version: latest
```

Comment:
If you have more MRs, there might be a collisition. What about sha?

Reply:
This is a keyword. And there will be not collision because the result is stored in the job as artifact.
https://github.com/plgd-dev/client-application/actions/runs/2582432619

Reply:
`latest` is the version of goreleaser

Reply:
![image](https://user-images.githubusercontent.com/23374974/176458003-eadba4b7-12ff-44b1-8a1f-fb45d219a6a5.png)